contracts/JBController.sol

```diff
@@ -477,28 +502,31 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
     returns (uint256 beneficiaryTokenCount)
   {
     // Can't send to the zero address.
-    require(_reservedRate == 10000 || _beneficiary != address(0), '0x2f: ZERO_ADDRESS');
+    if (_reservedRate != MAX_TOKEN_RATE || _beneficiary == address(0)) {
+      revert JBErrors.ZERO_ADDRESS();
+    }
 
     // There should be tokens to mint.
-    require(_tokenCount > 0, '0x30: NO_OP');
+    if (_tokenCount == 0) {
+      revert JBErrors.NO_OP();
+    }
 
     // Get a reference to the project's current funding cycle.
     JBFundingCycle memory _fundingCycle = fundingCycleStore.currentOf(_projectId);
 
     // If the message sender is not a terminal delegate, the current funding cycle must not be paused.
-    require(
-      !_fundingCycle.mintPaused() || directory.isTerminalDelegateOf(_projectId, msg.sender),
-      '0x31: PAUSED'
-    );
+    if (_fundingCycle.mintPaused() || !(directory.isTerminalDelegateOf(_projectId, msg.sender))) {
```

Comment:
This is not correct, and it seems to be a problem in almost all files (particularly untested ones).

You must negate operators as well. Should be &&. Also as mentioned in the other comment the negation should not be wrapped in parenthesis. Please fix everywhere.